Tests/LibGfx/TestImageWriter.cpp

```diff
@@ -387,6 +387,23 @@ TEST_CASE(test_tiff)
     // TRY_OR_FAIL((test_roundtrip<Gfx::TIFFWriter, Gfx::TIFFImageDecoderPlugin>(TRY_OR_FAIL(create_test_rgba_bitmap()))));
 }
 
+TEST_CASE(test_tiff_icc)
+{
+    auto sRGB_icc_profile = MUST(Gfx::ICC::sRGB());
+    auto sRGB_icc_data = MUST(Gfx::ICC::encode(sRGB_icc_profile));
```

Comment:
I see no reasons to use `MUST` here. I know this shouldn't fail, but a crashing test is always worse than a failing one, IMO. Especially as the crash occurs right here, rather than where it started to go wrong.

Reply:
/shruggie

It'll never fail in practice. These functions only fail on allocation failure. Maybe they should use MUST internally. Anyways, I changed it. It *does* look weird if parts of the test use MUST and parts use TRY_OR_FAIL :)